# #roll pitch yaw x y z joint
home1: [0, 90, 150, -0.98, -0.65, 0.26, 0]
home2: [80, 0, -107, -0.52, -0.72, 0.4, 0]
home2a: [80, 0, -107, -0.4, -0.72, 0.4, 0] 


# back1: [80, -2, -105, -0.65, -0.85, 0.63, 0] #ultima rot di 90
back1: [0, 90, 150, -0.65, -0.85, 0.63, 0] 
back2: [0, 0, 0, -0.3, -0.8, 0.3, 0]
# back1: [80, -2, -105, -0.65, -0.72, 0.72, 0]
# back2: [0, 0, 0, -0.2, -0.67, 0.63, 0]

first1: [0, 90, 130, -0.7, -0.7, 0.5, 0]
first2: [0, 0, 0, -0.3, -0.9, 0.4, 0]

start: [80, -2, -127, -0.98, -0.65, 0.20, 0] 
# start: [80, -2, -127, -0.98, -0.65, 0.26, 0] #ultima 
one:   [80, -6, -125, -0.89, -0.41, 0.2, 0]
two:   [76, 2, -80, -1, -0.73, 0.2, 0]

hand_port: /dev/ttyUSB1
